packages/manager/src/features/Linodes/LinodesCreate/VPCPanel.tsx

```diff
@@ -227,12 +226,11 @@ export const VPCPanel = (props: VPCPanelProps) => {
               selectedVPCId && selectedVPCId !== -1
                 ? vpcDropdownOptions.find(
                     (option) => option.value === selectedVPCId
-                  )
-                : null
+                  ) || null
```

Comment:
```suggestion
                  ) ?? null
```

It might be better to use the [null-coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) (`??`) here but either way is fine since `DropdownOption` can never be falsey anyways. Just a good habit I guess (since using `||` can lead to unexpected behavior when working with strings and numbers for example).

Reply:
done + ty! 🚀